feature/src/main/java/com/gdg/feature/calendar/CalendarRoute.kt

```diff
@@ -274,14 +274,15 @@ fun CalendarInfoBox(data: ScheduleEntity) {
                 color = CrowdZeroTheme.colors.gray600
             )
             Text(
-                text = data.jurisdiction,
+                text = data.jurisdiction.split("\n").joinToString(stringResource(R.string.calendar_jurisdiction_rest)),
```

Comment:
P2: 쉼표보다는 띄어쓰기로 표현하는게 더 나은 것 같슈
```suggestion
                text = data.jurisdiction.replace("\n", " "),
```

Reply:
처음에 띄어쓰기로 했었는데 뭔가 눈에 안 들어와서 쉼표로 바꿨거든? 한번 봐줄래?? 
![image](https://github.com/user-attachments/assets/9b9246cf-4599-45f5-9f90-c9d131df6551)

Reply:
남대문, 종 로 이게 낫다는거지??